lib/Interpreter/CppInterOpInterpreter.h

```diff
@@ -347,6 +347,10 @@ class Interpreter {
         const_cast<const Interpreter*>(this)->getDynamicLibraryManager());
   }
 
+  /// @brief As a interface to store paths added in AddIncludePaths
+  std::vector<std::string> include;
```

Comment:
Why do we need this. The include paths are stored in the HeaderSearch already.

Reply:
So we know that AddincludePaths() is fetching the paths and giving it to HeaderSearch, now GetIncludePaths() had the aim of exposing all the paths which were added(not present in HeaderSearch) to the user, for this we had to get/store all those paths which were being added to HeaderSearch with a vector ( as an interface) as AddIncludePaths() does'nt store any paths (makes it available for HeaderSearch).

Initially, I tried other methods to store all the paths from AddIncludePaths() itself but for that we require a change in function definition and would not require another function GetIncludePaths(). I also tried creating wrappers around AddIncludePaths() but to maintain the function signature I could'nt find a way to store the paths.